src/Storefront/Theme/ThemeMergedConfigBuilder.php

```diff
@@ -52,6 +52,8 @@ public function getPlainThemeConfiguration(string $themeId, Context $context): a
         $themes = $this->themeRepository->search($criteria, $context)->getEntities();
 
         $theme = $themes->get($themeId);
+        \assert($theme instanceof ThemeEntity);
+
         if (!$theme) {
```

Comment:
```suggestion
        if (!$theme instanceof ThemeEntity) {
```

if you want to have auto completion from the IDE, you can do this